ArduCopter/mode.cpp

```diff
@@ -27,6 +27,10 @@ Copter::Mode *Copter::mode_from_mode_num(const uint8_t mode)
             ret = &mode_auto;
             break;
 
+        case CHASE:
```

Comment:
I think that it is easier to understand how to add it after "case SMART_RTL" according to the order of enum definition control_mode_t.
There are parts that are different. . . . Recently, it seems that we are adding at the end.

Reply:
+1